spec/sections_spec.rb

```diff
@@ -19,11 +19,11 @@ class APage < SitePrism::Page
     end
 
     it 'should create matching object method' do
-      expect(page).to respond_to(:single_section)
+      expect(subject).to respond_to(:single_section)
     end
 
     it 'should create matching existence method' do
-      expect(page).to respond_to(:has_single_section?)
+      expect(subject).to respond_to(:has_single_section?)
```

Comment:
Maybe I'll look into a way to better implement/test these.  This only verifies the method is created, I think we rely entirely on cukes for validating the method behavior. 😞